webapi/src/main/scala/org/knora/webapi/responders/v2/ResourcesResponderV2.scala

```diff
@@ -1161,9 +1160,31 @@ class ResourcesResponderV2(responderData: ResponderData) extends ResponderWithSt
 
     Future
       .sequence(
-        listNodesThatShouldExist
-          .map(listNodeIri => ResourceUtilV2.checkListNodeExists(listNodeIri, appActor))
-          .toSeq
+        listNodesThatShouldExist.map { listNodeIri =>
+          for {
+            checkNode <- ResourceUtilV2.checkListNodeExistsAndIsRootNode(listNodeIri, appActor)
+
+            _ = checkNode match {
+                  // it doesn't have isRootNode property - it's a child node
+                  case Right(false) =>
+                    FastFuture.successful(())
+                  // it does have isRootNode property - it's a root node
+                  case Right(true) =>
+                    FastFuture.failed(
```

Comment:
I'm not sure if you need `FastFuture.failed` when you are throwing.

Reply:
That was the most tricky part. W/o`FastFuture.failed()` it didn't compile, but strangely now it seems working. Suspect metals was broken at that time...